scripts/create_project_info.py

```diff
@@ -1,3 +1,77 @@
-from version_utils import create_project_info
+import os
+import re
+import subprocess
+from datetime import datetime
+from string import Template
+
+
+def kolibri_version():
+    """
+    Returns the major.minor version of Kolibri if it exists
+    """
+    with open('./src/kolibri/VERSION', 'r') as version_file:
+        # p4a only likes digits and decimals
+        return version_file.read().strip()
```

Comment:
@benjaoming Is the VERSION file a reliable way of understanding Kolibri's current version?

Reply:
Not really. It's not a part of the repository, and it's not a guaranteed part of the running environment. It's a guaranteed part of the source distribution, though.

I would need to understand the intention a bit better, but I think this could be okay.

Reply:
It's only there to be an indicator of the version of Kolibri being used within the Android wrapper, for debugging purposes.